### PR TITLE
feat(backend): set a default custom user properties setting

### PR DESCRIPTION
**Pull request by asf-algoo**
#4157

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [X] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/development.ini.sample [hunk 1]
```diff
@@ -364,15 +364,10 @@ api.key = %(basic_setup.api_key)s
...
 # See the Tracim settings documentation (available in source code, see "user_custom_properties.md") for more
-# information about it. By default, Tracim doesn't use custom properties.
-; user.custom_properties.json_schema_file_path =
-; user.custom_properties.ui_schema_file_path =
-; user.custom_properties.translations_dir_path =
-## example with user.custom_properties.dir shortcut:
-# user.custom_properties.dir = %(here)s/examples/user_custom_properties/default
-# user.custom_properties.json_schema_file_path = %(user.custom_properties.dir)s/schema.json
-# user.custom_properties.ui_schema_file_path = %(user.custom_properties.dir)s/ui.json
-# user.custom_properties.translations_dir_path = %(user.custom_properties.dir)s/locale
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample:375]: I think we can save this exemple in the configuration file.

File: backend/development.ini.sample [hunk 2]
```diff
@@ -364,15 +364,10 @@ api.key = %(basic_setup.api_key)s
...
+# information about it.
+; user.custom_properties.json_schema_file_path = %(here)s/tracim_backend/templates/user_custom_properties/default/schema.json
+; user.custom_properties.ui_schema_file_path = %(here)s/tracim_backend/templates/user_custom_properties/default/ui.json
+; user.custom_properties.translations_dir_path = %(here)s/tracim_backend/templates/user_custom_properties/default/locale
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample:370]: We need docker modification in common.sh for this kind of path (`%(here)s` not work in docker context) and also information in changelog about what we need to do when upgrading instance (when development.ini already exist.

I can done docker modification if you want.

File: backend/development.ini.sample [hunk 3]
```diff
@@ -364,15 +364,10 @@ api.key = %(basic_setup.api_key)s
 
 ## User custom properties: This features allows defining custom instance-global fields on users of this instance.
 # See the Tracim settings documentation (available in source code, see "user_custom_properties.md") for more
-# information about it. By default, Tracim doesn't use custom properties.
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample:367]: Need changeor remove wording: ` By default, Tracim doesn't use custom properties.`

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]

File: backend/development.ini.sample (hunk 1 above)
Comment (raphj) [reply backend/development.ini.sample:375]: is this necessary, with the default values that it replaces and something like this being already in the documentation?

File: backend/development.ini.sample (hunk 3 above)
Comment (raphj) [reply backend/development.ini.sample:367]: This is removed, you are looking at a replaced line :-)

Comment (PhilippeAccorsi) [reply backend/development.ini.sample:367]: Good.

File: backend/development.ini.sample (hunk 2 above)
Comment (PhilippeAccorsi) [reply backend/development.ini.sample:370]: Done for docker.

**Comment by PhilippeAccorsi** [comment]
Need a manual test (also with docker image on new and existing installation).

**Review by grignards — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
I make test for this PR.

**Review by inkhey — APPROVED** [review]

**Review by grignards — APPROVED** [review]

**Review by PhilippeAccorsi — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
Tested on branch with 2 new docker images.

**Pull request merged**